### PR TITLE
Prepare 0.37.0 release

### PR DESCRIPTION
**Pull request by srenatus**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,128 @@
...
 
-## Unreleased
+## 0.37.0
+
+This release contains a number of fixes and enhancements.
+Please be aware that it also includes a security fix relevant for anyone
+using _optimized bundles_.
+
+This is the first release that includes a binary and a docker image for
+`linux/arm64`: `opa_linux_arm64_static` and `openpolicyagent/opa:0.37.0-static`.
```
Comment (johanfylling) [inline CHANGELOG.md:13]: Should that be `:`->`,`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,125 @@
...
+Thanks to @ngraef for contributing the build changes necessary.
+
+### Strict Mode
+
+There have been numerous checks in the compiler that fall into this category:
```
Comment (johanfylling) [inline CHANGELOG.md:18]: You could read this as if these checks have already been in effect in previous releases.
Consider `There have been numerous possible checks ..`, or similar.
No biggie, as any wrong assumptions are corrected shortly after.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,7 +3,125 @@
...
+- Bundles bug fix: Roundtrip manifest before hashing to allow changing the manifest
+  and still using signature verification of bundles ([#4233](https://github.com/open-policy-agent/opa/issues/4233)),
+  reported by @CristianJena
+
+- The test runner now also supports custom builtins, when invoked through the Golang
+  interface (authored by @MIA-Deltat1995)
+
+- The compile package and the `opa build` command support a new output format: "plan".
+  It represents a _query plan_, steps needed to take to evaluate a query (with policies).
+  The plan format is a JSON encoding of the intermediate representation (IR) previously
```
Comment (johanfylling) [inline CHANGELOG.md:52]: `previously`; it's not still used for that?

**Review by johanfylling — APPROVED** [review]
Looks great! Only nits.

File: CHANGELOG.md (hunk 3 above)
Comment (srenatus) [reply CHANGELOG.md]: Of course. thanks, dropped the word.

**Review by johanfylling — APPROVED** [review]

**Review by tsandall — APPROVED** [review]
LGTM. Plenty of great changes.

**Pull request merged**